tests/test_e2e.py

```diff
@@ -0,0 +1,31 @@
+import os
+import time
+
+import pytest
+import requests
+
+
+def test_upload_to_elastic(file_pdf_path, elasticsearch_storage_handler):
+    """
+    Given that I have removed all existing chunks from ES
+    When I POST a file to core-api/file
+    I Expect a Chunk with a non-null embedding ... eventually
+    """
+    for chunk in elasticsearch_storage_handler.read_all_items("Chunk"):
+        elasticsearch_storage_handler.delete_item(chunk.uuid, "Chunk")
+
+    file_name = os.path.basename(file_pdf_path)
+    files = {"file": (file_name, open(file_pdf_path, "rb"), "application/pdf")}
+    response = requests.post(url="http://localhost:5002/file", files=files)
```

Comment:
Do we have this url in env vars anywhere? Ignore if not

Reply:
good Q.... its specific to the integration test, if the tests were run inside a container then the host would be `elasticsearch` and maybe this should be the case?

Reply:
worth a ticket to decide how the tests should be run

Reply:
Agreed 👍